api/config.py

```diff
@@ -42,6 +42,7 @@
         'prefetch': False
     },
     'auth': {
+        'auth_type': 'google',
```

Comment:
@ryansanford let me know if you meant something other than this in your latest comment.